presto-yarn-package/src/main/resources/appConfig-test-no-catalog.json

```diff
@@ -17,7 +17,7 @@
     "site.global.jvm_args": "['-server', '-Xmx1024M', '-XX:+UseG1GC', '-XX:G1HeapRegionSize=32M', '-XX:+UseGCOverheadLimit', '-XX:+ExplicitGCInvokesConcurrent', '-XX:+HeapDumpOnOutOfMemoryError', '-XX:OnOutOfMemoryError=kill -9 %p']",
 
     "application.def": ".slider/package/PRESTO/${app.package.name}.zip",
-    "java_home": "/usr/lib/jvm/java"
```

Comment:
this needs to be changed because the new cdh5 docker images doesnt have this location and jdk1.8u92 is installed at /usr/java/. This is part of this PR because 0.148 requires jdk > u60